developerportal/settings/base.py

```diff
@@ -165,7 +165,7 @@
 EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
 EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER")
 EMAIL_PORT = int(os.environ.get("EMAIL_PORT", 587))
-EMAIL_USE_TLS = bool(os.environ.get("EMAIL_USE_TLS", True))
+EMAIL_USE_TLS = os.environ.get("EMAIL_USE_TLS", "True") == "True"
```

Comment:
This is cleaning up a bug where `bool()` doesn't correctly respect an env var being set to `False`, because `bool("False")` is truthy

Reply:
However if the env var is NOT set, then the default value was being used fine. The change covers the situation where we want to set it to `False` via env vars, which would have failed prior to this change